### ISSUE TITLE
\ncong and \overrightarrow not typeset properly

### ISSUE THREAD

**Issue by jano-m** [mathjax/MathJax#2497]
### Issue Summary

Mathjax typesets \ncong and \overrightarrow{ab} differently from Latex.
\ncong: in Latex whole \cong is crossed out, while in Mathjax only = is crossed out (which has different meaning)
\overrightarrow{ab}: the arrow above is short, equivalent to \vec{ab}, Latex would make the arrow span from left to right boundary of the (imaginary) box containing ab
Note: if the argument of the command gets longer, it will start working properly

Mathjax:
![image](https://user-images.githubusercontent.com/266826/90171525-7cdcea00-dd56-11ea-8cae-d5246f1e9c34.png)
Latex:
![image](https://user-images.githubusercontent.com/266826/90171586-8f572380-dd56-11ea-9205-f82b06a80994.png)

### Technical details:

* MathJax Version: 3.0.5
* Client OS: Windows 10
* Browser: Chrome 84.0.4147.125

**Comment by dpvc** [comment]
Thanks for the report.

The first issue (with `\ncong`) is due to a bad mapping of the glyphs in the original MathJax TeX fonts: U+2247 was inadvertently placed at U+2246, and `\ncong` was set to U+2246.  The font data for version 3 maps the glyph to U+2247 correctly, but we neglected to redefine `\ncong` to map to the correct location.  I have made a pull request to fix the issue.  In the meanwhile, you could add

``` html
<script>
MathJax = {
  startup: {
    ready() {
      if (MathJax.version === '3.0.5') {
        const MapHandler = MathJax._.input.tex.MapHandler.MapHandler;
        const Symbol = MathJax._.input.tex.Symbol.Symbol;
        MapHandler.getMap('AMSsymbols-mathchar0m0').add('ncong', new Symbol('nconfg', '\u2247'));
        MathJax.startup.defaultReady();
      }
    }
  }
};
</script>
```

before the script that loads the MathJax component you are using in order to fix the errant definition.

----

The second issue, with `\overrightarrow` is more significant.  MathJax will remap some accent characters to other ones (mostly for MathML input where people use the wrong characters frequently).  In particular, the right arrow U+2192 is remapped to the vector arrow U+20D7, so that `<mover><mi>x</mi><mo>&#x2192;</mo></mover>` will produce the output you would get from `\vec x`.  When the accent is stretchy, U+20D7 is converted back to U+2192, but it turns out that MathJax was applying the remapping again and it ended up U+20D7 after all.  I have submitted a pull request to fix that, but it is not so easy to patch it.  The straight-forward solution is to remove the remapping of the U+2192 so that neither remapping occurs.  That can be done with

``` html
<script>
MathJax = {
  startup: {
    ready() {
      if (MathJax.version === '3.0.5') {
        MathJax.startup.defaultReady();
        delete MathJax.startup.output.font.remapChars.accent[0x2192];
      }
    }
  }
};
</script>
```

Of course, you can combine these two into one configuration:

``` html
<script>
MathJax = {
  startup: {
    ready() {
      if (MathJax.version === '3.0.5') {
        const MapHandler = MathJax._.input.tex.MapHandler.MapHandler;
        const Symbol = MathJax._.input.tex.Symbol.Symbol;
        MapHandler.getMap('AMSsymbols-mathchar0m0').add('ncong', new Symbol('nconfg', '\u2247'));
        MathJax.startup.defaultReady();
        delete MathJax.startup.output.font.remapChars.accent[0x2192];
      }
    }
  }
};
</script>
```

**Comment by jano-m** [comment]
Thank you!

**Comment by dpvc** [comment]
Fixed in v3.1 released today.
